about.css

```diff
@@ -0,0 +1,1065 @@
+  padding: 15px;
+}
+
+.main-title {
+  color: #ec5242;
+  font-weight: bolder;
+  font-size: 2rem;
+}
+
+.title {
+  width: 100%;
+}
+
+.para2 {
+  color: var(--color-lightblack);
+  background-color: rgb(242, 242, 242);
+  display: flex;
+  border: 3px solid #fff;
+  align-items: center;
+  font-weight: 600;
+  line-height: 1.6rem;
+  margin-top: 25px;
+  padding: 1rem;
+}
+
+.year {
+  color: var(--color-black);
+  font-weight: 900;
+  font-size: 1.5rem;
+  margin: 1.5rem 0 0 24px;
+}
+
+.differ {
+  color: var(--color-lightblack);
+  margin: 10px 0 50px 24px;
+}
+
+.sec2 {
+  background-color: #272a32;
+  background-image: url(important-img/diamond.png);
+  width: 100%;
+  color: var(--color-white);
+  display: flex;
+  flex-direction: column;
+  padding: 30px 20px;
+}
+
+.sec2 .parr {
+  display: none;
+}
+
+.sec-2-title {
+  font-size: 30px;
+  align-self: center;
+}
+
+.guest img {
+  background-image: url(./important-img/image-bg.png);
+}
+
+.sec3 .speakers .guest img {
+  background-size: 120px 100px;
+  background-repeat: no-repeat;
+  width: 150px;
+  height: 150px;
+  object-fit: cover;
+  padding-left: 5%;
+  padding-top: 5%;
+}
+
+.line {
+  width: 2rem;
+  height: 1px;
+  border: 1px solid var(--color-secondaryDeep);
+}
+
+.program {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  gap: 20px;
+  margin-top: 1.6rem;
+}
+
+.list {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  align-items: center;
+  padding: 18px 8px 18px 14px;
+  gap: 12px;
+  background-color: rgba(255, 255, 255, 0.1);
+}
+
+.list i {
+  font-size: 30px;
+}
+
+.list-header {
+  color: var(--color-secondary);
+  flex: 4;
+}
+
+.list-para {
+  width: 100%;
+  flex: 5;
+  line-height: 1.6rem;
+}
+
+.join {
+  display: flex;
+  display: none;
+  background-color: var(--color-secondary);
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  text-align: center;
+  width: 70%;
+  padding: 1.6rem;
+  border: none;
+  margin: 30px;
+}
+
+.join h3 {
+  color: var(--color-white);
+  font-size: 20px;
+  word-spacing: 0.3rem;
+  padding: 8px;
+}
+
+.sec3 {
+  display: flex;
+  flex-direction: column;
+  padding: 50px;
+  color: var(--color-lightblack);
+}
+
+.sec3 .speakers {
+  margin-top: 40px;
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+}
+
+.sec3 .speakers .guest {
+  display: flex;
+  -moz-column-gap: 16px;
+  column-gap: 16px;
+}
+
+.sec3 .speakers .guest section {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+footer .foot-last h3 {
+  font-weight: lighter;
+  display: flex;
+  flex-direction: column;
+  width: 40%;
+  justify-content: center;
+  align-items: center;
+  font-size: 14px;
+  padding: 30px 0;
+}
+
+.sec3 .speakers .guest section h3 {
+  align-self: flex-start;
+}
+
+.sec3 .speakers .guest section .p1 {
+  width: 100%;
+  font-size: 15px;
+  color: var(--color-secondaryDeep);
+}
+
+.sec3 .speakers .guest section .p2 {
+  font-size: 13px;
```

Comment:
- [OPTIONAL] I notice that there are some styles you add in your `about.css` file and those are unused because you don't have those classes in your `about.html` file. Please kindly delete all the unused lines of code in this file and try to keep your file clean 😉 🙃